pom.xml

```diff
@@ -228,9 +214,7 @@
 				<module>stellaris</module>
 				<module>stringtemplate</module>
 				<module>suokif</module>
-				<module>swift2</module>
-				<module>swift3</module>
-				<module>swift-fin</module>
```

Comment:
This is incorrect, swift-fin is a completely different language, compare https://en.wikipedia.org/wiki/Society_for_Worldwide_Interbank_Financial_Telecommunication and https://en.wikipedia.org/wiki/Swift_(programming_language). So, `siwft-fin` should be returned back.

Reply:
oops